docs/pages/preview/eas-update/deployment-patterns.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Patterns to deploy your app
+---
+
+Once we've created features and fixed bugs, we want to get our changes out to our users as soon as we can, as safely as we can. Often "safe" and "fast" are opposing forces when delivering code to our users. We could push our code directly to production, which would be fast, but it would also be unsafe since we never tested our code. On the other hand, we could make test builds, share them with a QA team, and release periodically. That would be safer, but also slower to deliver changes to our users.
+
+Depending on your project, you'll have some tolerance for how "fast" and how "safe" you'll need to be when delivering updates to your users.
+
+There are a three parts that make up a successful deployment process that we'll need to consider:
+
+1. Creating builds
+   1. We can create builds for production use only.
+   2. We can create builds for production use and separate builds for testing use.
+2. Testing changes
+   1. We can test changes with TestFlight and Play Store Internal Track.
+   2. We can test changes with an internal distribution build.
+   3. We can test changes with Expo Go or a development app.
+3. Publishing updates
+   1. We can publish updates without considering branch names.
+   2. We can create update branches that are environment based, like `production` and `staging`.
```

Comment:
Everywhere else the names of branches and channels are quoted instead of code-formatted